test/linalg/generic.jl

```diff
@@ -342,3 +342,6 @@ Base.abs(a::ModInt{n}) where {n} = a
 Base.:<(a::ModInt{n}, b::ModInt{n}) where {n} = a.k < b.k
 
 @test A*(lufact(A, Val{true})\b) == b
+
+@test_throws ErrorException ctranspose(rand(2,2,2,2))
+@test_throws ErrorException transpose(rand(2,2,2,2))
```

Comment:
Are these checking that `[c]transpose` fails for rank greater than two? If so, perhaps an explanatory comment? :)